src/stores/DAOEntityStore.js

```diff
@@ -121,14 +121,36 @@ class DAOEntityStore<TEntity: { id: string }, TEntityMutator: {}> {
     this.entityItemsByID.get(id),
   );
 
-  getByQueryFilters = createTransformer(
-    (queryFilters: Array<QueryFilter>): Array<TEntity> =>
-      this.entityItemsByID
-        .values()
-        .filter((entityItem: TEntity): boolean =>
-          DAOApi.doesSatisfyToQueryFilters(entityItem, queryFilters),
-        ),
-  );
+  getByQueryOptions = createTransformer((queryOptions: QueryOptions): Array<
+    TEntity,
+  > => {
+    const { filters, orderBy } = queryOptions;
+    let entities = this.all;
+
+    if (filters) {
+      entities = entities.filter((entityItem: TEntity): boolean =>
+        DAOApi.doesSatisfyToQueryFilters(entityItem, filters),
```

Comment:
nit: `doesSatisfyToQueryFilters` seems weird.  Maybe `doesSatisfyQueryFilters`

Reply:
ok,  but its inside our api lib, so I'll have to switch there first.